zipkin/propagation.go

```diff
@@ -24,12 +24,21 @@ import (
 )
 
 // Propagator is an Injector and Extractor
-type Propagator struct{}
+type Propagator struct{
+	enableBaggage bool
+	baggagePrefix string
+}
 
 // NewZipkinB3HTTPHeaderPropagator creates a Propagator for extracting and injecting
 // Zipkin HTTP B3 headers into SpanContexts.
 func NewZipkinB3HTTPHeaderPropagator() Propagator {
-	return Propagator{}
+	return Propagator{enableBaggage: true, baggagePrefix: "baggage-"}
+}
+
+// NewZipkinB3HTTPHeaderPropagator creates a Propagator for extracting and injecting
```

Comment:
comment on exported function NewZipkinB3HTTPHeaderPropagatorWithBaggage should be of the form "NewZipkinB3HTTPHeaderPropagatorWithBaggage ..."